ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -446,36 +449,133 @@ def test_full_backup_reinstall_restore_with_vault(self):
 
 
 class TestBackupAndRestoreWithReplica(IntegrationTest):
-    """Regression test for https://pagure.io/freeipa/issue/7234"""
-    num_replicas = 1
+    """
+        Regression tests for issues 7234 and 7455
+
+        https://pagure.io/freeipa/issue/7234
+            - check that oddjobd service is started after restore
+            - check new replica  setup after restore
+        https://pagure.io/freeipa/issue/7455
+            check that after restore and replication reinitialization
+                - users and CA data are at state before backup
+                - CA can be installed on existing replica
+                - new replica with CA can be setup
+    """
+    num_replicas = 2
     topology = "star"
 
     @classmethod
     def install(cls, mh):
+        cls.replica1 = cls.replicas[0]
+        cls.replica2 = cls.replicas[1]
         if cls.domain_level is None:
             domain_level = cls.master.config.domain_level
         else:
             domain_level = cls.domain_level
 
-        if cls.topology is None:
-            return
-        else:
-            tasks.install_topo(
-                cls.topology, cls.master, [],
-                cls.clients, domain_level
-            )
+        # Configure only master and one replica.
+        # Replica is configured without CA
+        tasks.install_topo(
+            cls.topology, cls.master, [cls.replica1],
+            cls.clients, domain_level,
+            setup_replica_cas=False
+        )
+
+    def create_csr(self, host):
+        request_path = host.run_command(['mktemp']).stdout_text
```

Comment:
Are you sure that ``request_path`` does not contain a ``\n``? On other places, FreeIPA code strips off newlines.

Reply:
fixed